src/components/pages/global-values-page/components/values/GVItemsCustomOrder.tsx

```diff
@@ -61,8 +61,8 @@ export const GVItemsCustomOrder = () => {
                             data-movable-handle={true}
                             className={styleCustomOrder.itemDragHandle}
                         >
-                            <Up />
-                            <Down />
+                            <ChevronUpIcon />
+                            <ChevronDownIcon />
```

Comment:
Er det slik at disse ikke skal skjules for skjermleser?

Reply:
Disse skal eventuelt leses opp ja. Er bare til bruk i redaktørgrensesnittet (i Content Studio)